people/s5ehfr9/page.en.md

```diff
@@ -3,4 +3,18 @@ name: s5ehfr9
 info:
     alias: Lucy
     location: Mianyang, Sichuan
----
\ No newline at end of file
+---
+
+## Description 
+
+She has been particularly interested in aerospace and military since childhood, and loves to play various flight and war simulation games. She also has a strong interest in game development, enjoys writing mods for the games she loves, and has even participated in the development of independent games. Besides, she is also an excellent drummer and has played in bands, and she is also good at English, painting and so on.  
+
+But it was her who looked excellent in everyone's eyes, the MtF thoughts that had been suppressed in her heart since she was a child exploded completely, and she suffered from severe depression from then on.  
+
+With the ensuing feminization transformation, she was still unable to accept her own appearance. She dreams of being a cat with a lovely appearance and a free personality.  
+
+In the last few months, she was separated from her loving family, which made her suddenly relieved, and she could vent her emotions, but she also slowly allowed herself to degenerate. She found that she slowly lost interest in the things she once loved, and finally couldn't even concentrate on doing anything. At the same time, her mental illness began to spiral out of control, with acute anxiety outbreaks occurring very frequently. All of this made her blame herself even more.  
```

Comment:
In the last few months of her life, she was separated from her loving family. This made her suddenly feel relieved. She finally could vent her emotions, but also slowly allowed herself to degenerate. She found that she slowly lost interest in the things she once loved, and eventually couldn't even concentrate on doing anything. At the same time, her mental illness began to spiral out of control, with acute anxiety outbreaks occurring very frequently. All of this made her blame herself even more.